### PR TITLE
update the doc link in internal_arthitecture.md

### PR DESCRIPTION
**Pull request by showuon**
1. Update the configuration link from the old https://prometheus.io/docs/prometheus/latest/configuration/configuration/#%3C**xx_config**%3E  to https://prometheus.io/docs/prometheus/latest/configuration/configuration/#**xx_config**

2. Update the db.go link
Original
> Prometheus's local on-disk time series database is a [light-weight wrapper](https://github.com/prometheus/prometheus/blob/v2.3.1/storage/tsdb/tsdb.go#L102-L106) around [`github.com/prometheus/prometheus/tsdb.DB`](https://github.com/prometheus/prometheus/blob/master/tsdb/db.go#L92-L117).

But the db.go link (L92-L117) doesn't make any sense to the context. I think it should link to the db.go file directly. What do you think?

Signed-off-by: Luke Chen <showuon@gmail.com>

### CODE REVIEW

**Comment by brian-brazil** [comment]
> I think it should link to the db.go file directly. What do you think?

Yeah that's outdated. Just a link to db.go would be enough, we don't need the rest as that's all historical and no longer relevant.

**Comment by showuon** [comment]
> Yeah that's outdated. Just a link to db.go would be enough, we don't need the rest as that's all historical and no longer relevant.

Thank you, @brian-brazil

**Comment by showuon** [comment]
Just updated. Please check again @brian-brazil . Thanks.

File: documentation/internal_architecture.md [hunk 1]
```diff
@@ -93,15 +93,15 @@ Currently rules still read and write directly from/to the fanout storage, but th
...
+Prometheus's local on-disk time series database is a [light-weight wrapper](https://github.com/prometheus/prometheus/blob/v2.3.1/storage/tsdb/tsdb.go#L102-L106) around [`github.com/prometheus/prometheus/tsdb.DB`](https://github.com/prometheus/prometheus/blob/master/tsdb/db.go). The wrapper makes only minor interface adjustments for use of the TSDB in the context of the Prometheus server and implements the [`storage.Storage` interface](https://github.com/prometheus/prometheus/blob/v2.3.1/storage/interface.go#L31-L44). You can find more details about the TSDB's on-disk layout in the [local storage documentation](https://prometheus.io/docs/prometheus/latest/storage/).
```
Comment (brian-brazil) [inline documentation/internal_architecture.md:96]: You can remove all the light-weight wrapper stuff, that's all outdated and the code it's pointing to is no longer relevant.

**Comment by showuon** [comment]
Thanks @brian-brazil , I've removed the light-weight wrapper stuff. Now, it'll look like this:
image

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**